packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -52,7 +51,7 @@ export class IpfsTopology {
   ) {}
 
   async forceConnection(): Promise<void> {
-    const base: string[] = BASE_BOOTSTRAP_LIST[this.ceramicNetwork] || [];
+    const base: Multiaddr[] = BASE_BOOTSTRAP_LIST[this.ceramicNetwork] || [];
```

Comment:
Is there a change needed to `BASE_BOOTSTRAP_LIST` as well?

Reply:
Probably shouldn't be so restrictive, opened an issue @ https://github.com/ipfs/js-ipfs/issues/3638